broccoli/ember-tests-es.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+/* eslint-env node */
+
+const Funnel = require('broccoli-funnel');
+
+module.exports = function emberTestES(name) {
+  return new Funnel(`packages/${name}/tests`, {
+    getDestinationPath(relativePath) {
```

Comment:
I think this can be changed to avoid requiring us to stat and symlink all the files:

```js
return new Funnel(`packages/${name}/tests`, {
  destDir: `${name}/tests/`
});
```